inference/inference.py

```diff
@@ -322,6 +322,23 @@ def autoregressive_inference(params, ic, valid_data_full, model):
             }
           )
               
+    # inspect snapshot times at 5-days, 10-days.
+    # N.B. the predictions are 1-indexed, i.e. index 0 is the first prediction.
+    snapshot_timesteps = [(24 // 6 * k - 1, f"{k}-days") for k in [5, 10]]
```

Comment:
The first index in the metrics actually corresponds to the initial condition, i.e. you'll find RMSE[0] = 0. So don't think we want that minus 1 here.